charts/merlin/values.yaml

```diff
@@ -216,6 +216,14 @@ ui:
 service:
   externalPort: 8080
   internalPort: 8080
+serviceAccount:
+  # Specifies whether a ServiceAccount should be created
+  create: true
+  # The name of the ServiceAccount to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: merlin
+  annotations:
+    example.io/example: example
```

Comment:
I think it's better to specify this as a comment else this will be the default annotation when the helm chart is installed.

Reply:
Should be commented out. Doing

Reply:
commented.